numba/cuda/compiler.py

```diff
@@ -293,13 +291,15 @@ def inspect_ptx(self, args, nvvm_options={}):
         ----------
         args: tuple[Type]
             Argument types.
-        nvvm_options : dict; optional
-            See `CompilationUnit.compile` in `numba/cuda/cudadrv/nvvm.py`.
 
         Returns
         -------
         ptx : bytes
         """
+        if nvvm_options:
+            msg = 'nvvm_options kwarg for inspect_ptx is deprecated'
+            warn(msg, category=NumbaDeprecationWarning)
```

Comment:
Want to add this to the deprecation notice doc?

Reply:
I decided that deprecating it was a mistake (from the commit message un-deprecating it):

```
The idea behind deprecating this was to prevent a mismatch between
compiled code that is executed and code produced by the `inspect_ptx()`
function - however, as device functions aren't materialized standalone
(they are always part of a kernel), there is no risk of this mismatch.
Instead, `inspect_ptx()` stands as a useful mechanism for obtaining PTX
code for a device function if required - noting that `compile_ptx()`
does provide a better interface, so this whole function may be
deprecated / removed in future.
```